<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="addPartyNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="addPartyLeaderComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>90</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueen nimi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueenjohtaja</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPartyCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addPartyAddPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisää</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addPartyTypeComboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueen tyyppi</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addPartyNameLineEdit</tabstop>
  <tabstop>addPartyLeaderComboBox</tabstop>
  <tabstop>addPartyCancelPushButton</tabstop>
  <tabstop>addPartyAddPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
